streaming/src/main/scala/org/apache/spark/streaming/scheduler/ExecutorAllocationManager.scala

```diff
@@ -152,32 +150,35 @@ private[streaming] class ExecutorAllocationManager(
   private def validateSettings(): Unit = {
     require(
       scalingIntervalSecs > 0,
-      s"Config $SCALING_INTERVAL_KEY must be more than 0")
+      s"Config ${STREAMING_DYN_ALLOCATION_SCALING_INTERVAL.key} must be more than 0")
```

Comment:
Can we move these simple checks for a positive value to the new config objects you add here?

Reply:
Sure, I will update.